File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.cs
public class Solution {
    public string solution(string s) {
        string answer = "";
        string prev = "";
        
        foreach(var i in s){
            if(answer.Length == 0){
                answer += i.ToString().ToUpper();
            }
            else if(prev == " "){
                answer += i.ToString().ToUpper();
            }
            else {
                answer += i.ToString().ToLower();
            }
            
            prev = i.ToString();
        }
        return answer;
    }
}

File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.java
class Solution {
    public String solution(String s) {
        String answer = "";
        char prev = ' ';
        char[] arr = s.toCharArray();
        for(char i : arr){
            if(answer.length() == 0){
                answer += String.valueOf(i).toUpperCase();
            }
            else if(prev == ' '){
                answer += String.valueOf(i).toUpperCase();
            }
            else {
                answer += String.valueOf(i).toLowerCase();
            }

            prev = i;
        }
        return answer;
    }
}